### PR TITLE
[IMP] web: hide search panel when no records

### PR DESCRIPTION
**Pull request by sabr-odoo**
SPECIFICATION:

The search panel should unfold when records exist and fold when there are not.

Task-4009115

### CODE REVIEW

File: addons/web/static/src/search/search_panel/search_panel.js [hunk 1]
```diff
@@ -315,6 +315,9 @@ export class SearchPanel extends Component {
     }
 
     updateActiveValues() {
+        if (this.sections.length === 0) {
```
Comment (bit-odoo) [inline addons/web/static/src/search/search_panel/search_panel.js:318]: Hello @sabr-odoo I think we should fix this when the `sidebarExpanded` opens.

**Comment by bit-odoo** [comment]
Hello @sabr-odoo can you please add the test?

**Comment by sabr-odoo** [comment]
Hello @bit-odoo ,

I have added the test case.

Thank you :slightly_smiling_face:

**Review by aab-odoo — APPROVED** [review]
robodoo r+

**Comment by aab-odoo** [comment]
robodoo r+

**Review by aab-odoo — APPROVED** [review]
robodoo r+

**Pull request closed without merging**